de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/grid/GridViewDisplayStrings.properties

```diff
@@ -26,3 +26,4 @@ md5Checksum = Md5 Checksum
 checksum = Checksum
 metadataSaveError = Please fix all error(s) before saving. Attribute(s) cannot be empty!
 dataLinkTitle = Public Data Link
+copyMetadataPrompt = Select file(s) and folder(s) to which metadata will be copied to.
```

Comment:
I think you can drop the final `to` in this sentence: so `Select file(s) and folder(s) to which metadata will be copied.`